crates/wysiwyg/src/tests/test_paragraphs.rs

```diff
@@ -429,3 +429,18 @@ fn pressing_enter_at_the_start_of_a_multiline_block_quote() {
         "<blockquote><p>&nbsp;</p><p>|line_1</p><p>line_2</p></blockquote>"
     )
 }
+
+#[test]
+fn pressing_enter_in_the_middle_of_a_multiline_code_block() {
+    let mut model = cm("<pre>\n\n|line_1\nline_2</pre>");
+    model.enter();
+    assert_eq!(tx(&model), "<pre>&nbsp;\n\n|line_1\nline_2</pre>")
+}
```

Comment:
not sure about this one? shouldn't it be `<pre>\n|line_1\nline_2</pre>` the initial condition?

Reply:
The first new line char will be removed when parsing, so it actually only has a single line break.

Reply:
I could add a `&nbsp;` at the start of the code block instead of the double line break though.

Reply:
oh yeah right, actually maybe the other test is way more clear, the one I wrote below

Reply:
Done in [`86d17d4`](https://github.com/matrix-org/matrix-rich-text-editor/pull/531/commits/86d17d4de9cf92c810eef071d5bc03157e70225b).